var/spack/repos/builtin/packages/prng/package.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Prng(AutotoolsPackage):
+    """Pseudo-Random Number Generator library."""
+
+    homepage = "http://statmath.wu.ac.at/prng/"
+    url      = "http://statmath.wu.ac.at/prng/prng-3.0.2.tar.gz"
+    list_url = "http://statmath.wu.ac.at/prng"
```

Comment:
Does this `list_url` do anything? I think Spack already automatically checks the directory containing the `url`.

Reply:
I wasn't sure whether I should put it or not, just did it because it was requested in a different PR. I also think Spack is able to find out other versions from `url` though. When should it be added?

Reply:
It should be added when the page that lists downloads is different from the dirname of the URL.